_bookdown.yml

```diff
@@ -28,15 +28,15 @@ rmd_files:
 
   # End files
   - appendix.Rmd
-  - glossary/glossary.md
+  - chapters/solutions.Rmd
   - chapters/objectives.Rmd
   - chapters/keypoints.Rmd
-  - chapters/solutions.Rmd
-  - chapters/anaconda.Rmd
   - chapters/tree.Rmd
```

Comment:
I think style comes before tree?

Reply:
There was a random/unnecessary reference to style in the project structure section, so I took it out and added a reference in the scripting chapter instead, which is where we first write some Python code and is thus the most appropriate place for the initial reference to that Appendix to be.

Reply:
Got it, I should really stop trying to review when I'm in the middle of a hairy PR of my own.